resources/main.tf

```diff
@@ -115,3 +117,47 @@ resource "local_file" "values" {
   content  = "${data.template_file.values.rendered}"
   filename = "${path.module}/.helm-config/${terraform.workspace}/values.yaml"
 }
+
+resource "aws_iam_user" "concourse-user" {
+  name = "concourse-user"
+  path = "/tools/concourse/"
+}
+
+resource "aws_iam_access_key" "iam_access_key" {
+  user = "${aws_iam_user.concourse-user.name}"
+}
+
+data "aws_iam_policy_document" "policy" {
+  statement {
+    actions = [
+      "s3:CreateBucket",
+    ]
+
+    resources = [
+      "arn:aws:s3:::*",
+    ]
+  }
+
+  statement {
+    actions = [
+      "iam:CreateUser",
+    ]
+
+    resources = [
+      "arn:aws:iam::${data.aws_caller_identity.current.account_id}:*",
+    ]
+  }
+}
+
+resource "aws_iam_policy" "policy" {
+  name        = "concourse-account-policy"
```

Comment:
Same as above, replace `concourse-account` in the `name` and `description` fields with `${terraform.workspace}-concourse-user` for consistency.